docs/test-content.md

```diff
@@ -0,0 +1,152 @@
+# Test Content
+
+## serviceAckResponse c2d messages
+
+`serviceAckResponse` messages are used to acknowledge events that the device app is unable to observe.
+A `serviceAckId` could represent a telemetry message, a reported property write, or any other operation that needs to be verified by the service.
+The `serviceAckId` is created by the device and passed to the service app as part of an operation.
+When the service app verifies that operation, it returns the `serviceAckId` to the device inside a `serviceAckResponse` c2d message.
+When the device receives an operation's `serviceAckId` back, it knows that the operation has been verified by the service app.
+Because c2d is a fairly expensive operation, the service app gathers `serviceAckIds` in batches to send to the device app.
+
+```json
+  {
+    "thief": {
+      "cmd": "serviceAckResponse",
+      "serviceRunId": "ca3ceadb-ac57-4cca-8f9b-c5d1e4dc189d",
+      "pairingId": "3887e97e-6f06-484b-a5bf-1753913866d3",
+      "serviceAcks": [
+        {
```

Comment:
Why have these be an array of objects, instead of just an array of strings? The key for this outer object is `serviceAcks` so I don't see a value in the key `serviceAckId` before each value.

Reply:
there used to be extra properties with each ack.  flattening like you suggest is a good idea.